config/main.py

```diff
@@ -1548,6 +1548,34 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, force, file_form
             click.echo("Input {} config file(s) separated by comma for multiple files ".format(num_cfg_file))
             return
 
+    for inst in range(-1, num_cfg_file-1):
+        # Get the namespace name, for linux host it is None
+        if inst == -1:
+            namespace = None
+        else:
+            namespace = "{}{}".format(NAMESPACE_PREFIX, inst)
+
+        # Get the file from user input, else take the default file /etc/sonic/config_db{NS_id}.json
+        if cfg_files:
+            file = cfg_files[inst+1]
+        else:
+            if file_format == 'config_db':
+                if namespace is None:
+                    file = DEFAULT_CONFIG_DB_FILE
+                else:
+                    file = "/etc/sonic/config_db{}.json".format(inst)
+            else:
+                file = DEFAULT_CONFIG_YANG_FILE
+
+        # Check the file is properly formatted before proceeding.  
+        try:
```

Comment:
What is the behavior if config file doesn't exist (`os.path.exists(file) == false`)? 

Line 1608 - 1611 indicates instead of exiting, we skip the current namespace and check the next one?

Reply:
Updated the function so it skips the current namespace if the file doesn't exist